### ISSUE TITLE
How to evaluate the RandLA-Net on Toronto 3D per class with pytorch model?

### ISSUE THREAD

**Issue by Hao-HUST** [intel-isl/Open3D-ML#259]
Hi, 

I noticed that you provided the pre-trained torch version RandLA-Net Model on dataset Toronto 3D, and the mean IOU is 71.2, as you described.

However,  the accuracy and mean IOU are not calculated when I tried to run the pipeline.run_test() (torch version). Do you know how to calculate the mean IOU and per class IOU for Toronto 3D?

**Comment by sanskar107** [comment]
@Hao-HUST you can use the latest `dev` branch. We've added the IoU calculation for test set.
